src/index.ts

```diff
@@ -198,36 +194,89 @@ export default class MondayConnector extends BaseHttpConnector<
     return handled
   }
 
+  async query(
+    qs: string,
+    variables?: Record<string, any>,
+  ): Promise<any> {
+    const res = await this._sdk.api(qs, variables ? { variables } : undefined)
+    if ('data' in res) {
+      return res.data
+    }
+    throw new MondayError(res.errors)
+  }
+
+  columnValuesToObject(
+    columnValues: { title: string, value: string }[],
+  ): Record<string, any> {
+    const obj: Record<string, any> = {}
+    for (const cv of columnValues) {
+      obj[cv.title] = JSON.parse(cv.value)
+    }
+    return obj
+  }
+
   getBoard(boardIds: number | number[]): Promise<MondayApiReponse['data']> {
-    return this._api(GET_BOARD_QUERY, { variables: { board_ids: boardIds } })
+    return this.query(GET_BOARD_QUERY, { board_ids: boardIds })
+  }
+
+  async getBoardByName(name: string): Promise<number> {
+    const res = await this.query('query { boards { id name } }')
+    const board = (res?.boards as any).filter((b) => b.name === name)[0]
+    return board && parseInt(board.id, 10)
```

Comment:
I think either we rename the function to getBoard**Id**ByName() or the function needs to return the board itself, not only the id.
Second option is probably the best as from a user perspective, you need the board. And so you don't have to call getBoard with this id just after to get what you need